cds/modules/redirector/views.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of CERN Document Server.
+# Copyright (C) 2017 CERN.
+#
+# CERN Document Server is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# CERN Document Server is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with CERN Document Server; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""CDS redirector views."""
+
+from __future__ import absolute_import, print_function
+
+from cds.modules.deposit.api import Video
+from cds.modules.records.providers import CDSReportNumberProvider
+from flask import Blueprint, redirect, url_for
+
+
+blueprint = Blueprint(
+    'cds_redirector',
+    __name__,
+    template_folder='templates',
+    static_folder='static',
+)
+
+
+# /record/<pid_value>/embed/<filename>
+# /video/<report_number>/
+@blueprint.route('/video/<report_number>')
+def video_embed_alias(report_number):
+    """Redirect from the old video embed URL to the new one."""
+    pid = CDSReportNumberProvider.get(report_number).pid
+    recid = Video.get_record(pid.object_uuid)['recid']
```

Comment:
Probably you can get this value from the PID Store, it might be more efficient ...